module3/projects/rancid_tomatillos/technical_requirements.md

```diff
@@ -0,0 +1,237 @@
+---
+title: Rancid Tomatillos Technical Requirements
+module: 3
+tags: react,  testing, javascript, api, cypress
+---
+
+_[Back to Rancid Tomatillos Home](./index)_
+
+## Styling
+
+You do not have to match our styling for this project. That said, we want you to make intentional styling choices and for your final product to look polished. If you'd rather not spend time creating a design for this app, you may choose to match our design exactly. Pay close attention to the details - intentional hover states, spacing, and borders can make a huge difference. Be prepared to defend your styling choices in your evaluation with your instructor.
+
+## Iterations
+
+<section class="dropdown">
+### Iteration 0: Set Up & Planning
+
+You will be starting with this boilerplate repo: **[https://github.com/turingschool-examples/rancid-tomatillos-starter](https://github.com/turingschool-examples/rancid-tomatillos-starter)**. Follow the steps in the README carefully.
+
+
+**By the end of the day, Day 1:**
```

Comment:
Are we expecting students to send us this stuff? If not, I think we should mention that so that we don't get a bunch of DMs. Or we can be explicit about which of these we want "delivered" and which are benchmarks